### PR TITLE
scan: report status & wire up request index page 

### LINKED ISSUE #167: Data request page: Data report status

### LINKED ISSUE #181: Implement hook to get data request information by id

### LINKED ISSUE #216: Wire up request index page

### PR DESCRIPTION
**Pull request by evilpeach**
Fixes #167 
Fixes #181 
Fixes #216

### CODE REVIEW

File: scan/src/rpc/ScriptHook.re [hunk 1]
```diff
@@ -16,18 +16,29 @@ module Script = {
...
   };
 
   let decodeResultScript = json =>
     JsonUtils.Decode.{
-      name: json |> at(["info", "name"], string),
-      codeHash: json |> at(["info", "codeHash"], string) |> Hash.fromBase64,
-      params: json |> at(["info", "params"], list(decodeField)),
-      dataSources: json |> at(["info", "dataSources"], list(decodeField)),
-      creator: json |> at(["info", "creator"], string) |> Address.fromBech32,
+      info: json |> ScriptInfo.decode("info"),
```
Comment (sorawit) [inline scan/src/rpc/ScriptHook.re]: Let's do `json |> field("info", ScriptInfo.decode)` and make `ScriptInfo.decode` just takes `json`

File: scan/src/components/ReportTable.re [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+         <TBody key={txHash |> Hash.toHex} height=100>
+           <Row alignItems=Css.flexStart>
+             <Col> <div className=Styles.txhash /> </Col>
+             <Col size=1.0> <TElement elementType={txHash->TElement.Hash} /> </Col>
+             <Col size=0.35> <TElement elementType={reportedAtHeight->TElement.Height} /> </Col>
+             <Col size=0.4> <TElement elementType={reportedAtTime->TElement.Timestamp} /> </Col>
+             <Col size=1.0>
+               <TElement elementType={reporter->TElement.Address} />
+               <VSpacing size=Spacing.sm />
+               <TElement elementType={"(CoinGecko DataProvider)"->TElement.Detail} />
```
Comment (sorawit) [inline scan/src/components/ReportTable.re:46]: Remove this.

**Review by sorawit — CHANGES_REQUESTED** [review]
Please also show request result

File: scan/src/components/ReportTable.re (hunk 2 above)
Comment (evilpeach) [reply scan/src/components/ReportTable.re:46]: I'll make sure this part is valid in #223

**Review by sorawit — APPROVED** [review]

File: scan/src/components/ReportTable.re [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  open Css;
+
+  let txhash = style([marginLeft(`px(20))]);
+};
+
+[@react.component]
+let make = (~reports: list(RequestHook.Report.t)) => {
+  <>
+    <THead>
+      <Row>
```
Comment (prin-r) [inline scan/src/components/ReportTable.re:11]: Add Link to tx index page

File: scan/src/pages/RequestIndexPage.re [hunk 4]
```diff
@@ -32,99 +32,178 @@ module Styles = {
 
   let checkLogo = style([marginRight(`px(10))]);
 
+  let dataContainer =
+    style([display(`flex), border(`px(1), `solid, `hex("EEEEEE")), flexDirection(`column)]);
```
Comment (prin-r) [inline scan/src/pages/RequestIndexPage.re]: EEEEEE -> Colors.lightGray

**Review by prin-r — APPROVED** [review]

**Review by sorawit — APPROVED** [review]

**Pull request merged**